doc/porter-chart.md

```diff
@@ -0,0 +1,295 @@
+default       nginx-service   LoadBalancer   10.100.5.90   192.168.3.100   8088:32063/TCP           50s
+```
+
+## Visit nginx service
+
+Execute commands on linux:
+
+```bash
+$ curl 192.168.3.100:8088
+```
+
+# BGP mode
+
+## pre-requisites
+
+- Requires Kubernetes `1.17.3` or above.
+
+- Require `router` to start BGP mode.We will install bird on Centos7 system and use bird to implement BGP routing function. We call this machine `router`.
+
+- A linux machine, used to detect LoadBalancer of nginx
+
+## Network diagram
+
+```bash
+ ________________             ________________              ________________
+|               |            |                |            |                | 
+| k8s cluster   | <--------- |     router     | <--------- |   other host   |
+|_______________|            |________________|            |________________|
+```
+
+- We use bird to implement BGP mode on centos7 system.
+
+- Other hosts send packets to a `router`, and the `router` is sending packets to the k8s cluster.
+
+- The k8s cluster needs to use BGP to establish a connection with the `router`, so the `as`  of the two must be different.
+
+## configure on router
+
+Install bird on `router`:
+
+```bash
+$ yum install bird 
+$ systemctl enable bird
+```
+
+Configure BGP on the `router` as follows:
+
+```bash
+cat /etc/bird.conf
+protocol kernel {
+    scan time 60;       # Scan kernel routing table every 20 seconds
+    import none;        # Default is import all
+    export all;         # Default is export none
+    merge paths on;     # Enable ECMP, this parameter requires at least bird 1.6
+}
+
+protocol device {
+    scan time 10;       # Scan interfaces every 10 seconds
+}
+
+protocol static {
+}
+
+protocol bgp mymaster {   
+    description "10.55.0.127";                  # local ip
+    local as 65001;                             # local as.It must be different from the as of the port-manager
+    neighbor 10.55.0.124 port 17900 as 65000;   # Master node IP and AS number
+    source address 10.55.0.127;                 # Router IP 
+    import all; 
+    export all;
+    enable route refresh off;
+    add paths on;
+}
+```
+
+Start bird on the `router` and set ipv4 forwarding:
+
+```bash
+$ systemctl restart bird
+$ sysctl -w net.ipv4.ip_forward=1
+```
+
+Check whether the configuration takes effect on the `router`, you will see `mymaster` rule.
+
+```bash
+$ birdc show protocol
+BIRD 1.6.8 ready.
+name     proto    table    state  since       info
+kernel1  Kernel   master   up     18:01:55    
+device1  Device   master   up     18:01:55    
+static1  Static   master   up     18:01:55    
+mymaster BGP      master   start  18:01:55    Active        Socket: Connection refused
+```
+
+
+## Establish BGP connection on porter and router
+
+Execute commands on Kubernetes cluster:
+
+```bash 
+$ cat << EOF > bgp.yaml
+apiVersion: network.kubesphere.io/v1alpha1
+kind: Eip
+metadata:
+    name: eip-sample-pool
+spec:
+    # 修改ip地址段为实际环境的ip地址段。
```

Comment:
There are still some Chinese annotations in this section, please help to translate them into English.